importer/test_data/struct_expression.expected.asm

```diff
@@ -0,0 +1,10 @@
+mod other {
+    enum E1 {
+        A,
+    }
+    struct S1 {
+        a: int,
+        b: other::E1,
+    }
+}
+let s1 = other::S1{ a: 1, b: other::E1::A };
```

Comment:
Please also add a line that uses `Option` without explicitly importing it from the prelude.

Reply:
It seems that `prelude` is not included in these test cases. So I had to define a small fake std on top of the test.

Reply:
No, please don't do this, it does not test what we wanted to test here. If it doesn't work as an importer test, then instead please use a pipeline test.

Reply:
Or you can call `add_std` in the test function like in `load_dependencies_and_resolve'.

The point of testing the prelude is that it should work even without explicit import and without adding `std::prelude`.